autosynth/change_pusher.py

```diff
@@ -159,7 +167,7 @@ def push_changes(
             # Do a git dance to construct a branch with the commits squashed.
             temp_branch = str(uuid.uuid4())
             executor.check_call(["git", "branch", "-m", temp_branch])
-            executor.check_call(["git", "checkout", "master"])
+            executor.check_call(["git", "checkout", self.default_branch])
```

Comment:
We can leverage the `commit_count` to completely ignore the name of the default branch, like this:
https://github.com/googleapis/synthtool/compare/master...SurferJeffAtGoogle:no-master?expand=1#diff-14e3db0fd118f42dda8afc3bdf3d61f64370fc5c1a5ef34975d441337c413699

Reply:
Done